cluster/src/dunit/scala/io/snappydata/metrics/SnappyMetricsSystemDUnitTest.scala

```diff
@@ -60,16 +53,15 @@ class SnappyMetricsSystemDUnitTest(s: String)
         s" -peer-discovery-port=$port -client-port=$netPort")
     pw.close()
     val pw1 = new PrintWriter(new File(s"$confDir/leads"))
-    pw1.write(s"localhost -locators=localhost[$port] -dir=$snappyProductDir/work/lead1\n")
-    pw1.write(s"localhost -locators=localhost[$port] -dir=$snappyProductDir/work/lead2")
+    pw1.write(s"localhost -locators=localhost[$port] " +
```

Comment:
Why specify `spark.ui.port` for each lead? Does it cause any conflict?

Reply:
yes its causing conflict and i am getting bind exception So i added this spark.ui.port for getting metrics on that port